packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx

```diff
@@ -71,7 +70,7 @@ const storybookMeta: Meta<typeof Banner> = {
     severity: {
       options: bannerSeverityValues,
       control: { type: "radio" },
-      description: "Determine the color and icon of the alert",
+      description: "<b>Required.</b> Determine the color and icon of the alert",
```

Comment:
I thought there was a way to get Storybook to put a red asterisk. We might need to check their source code.

I know you put this in, but I don't wanna add `<b>Required.</b>` each time. It doesn't scale. Can we remove those?

Reply:
I think I found it:
![image](https://github.com/okta/odyssey/assets/97464104/fdd11884-10ad-4f23-874b-806c812b3aea)

Source:
https://storybook.js.org/docs/vue/api/arg-types

Reply:
Updated